installtests/centos.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-set -ex  # Exit on any command failure
+set -x  # Exit on any command failure
```

Comment:
Not being difficult, but what's `-u` do? I honestly can't remember and googling for bash command line switches is never fun. =)